src/engraving/libmscore/engravingitem.cpp

```diff
@@ -176,6 +177,18 @@ AccessibleItemPtr EngravingItem::createAccessible()
     return std::make_shared<AccessibleItem>(this);
 }
 
+void EngravingItem::notifyAboutNameChanged()
+{
+    if (!selected()) {
+        return;
+    }
+
+    if (m_accessible) {
+        doInitAccessible();
+        m_accessible->accessibleRoot()->notifyAboutFocuedElemntNameChanged();
```

Comment:
"notifyAboutFocuedElemntNameChanged"?  Is that the name of one of our functions?  Two typos for the price of one :-)

Reply:
Ah, I was rushing